torch/_inductor/codecache.py

```diff
@@ -555,7 +567,7 @@ def __bool__(self):
 
 invalid_vec_isa = InvalidVecISA()
 supported_vec_isa_list = [VecAVX512(), VecAVX2()]
```

Comment:
consider rename it to `supported_x86_vec_isa_list` to be more accurate?

Reply:
Yes, I can do that.

Reply:
Seems quite a few CI failures. Please check.

Reply:
Yes, I missed updates in the test path and in cases involving other paths of invocation. Reworking.